src/virtual_machine.rs

```diff
@@ -1,6 +1,12 @@
 // 2^16. 65536 locations.
 const MEMORY_MAX: usize = 2_usize.pow(16);
 
+const OPSIZE: u16 = 4;
+const ARGSIZE: u16 = 12;
+
+// Bitwise AND the instruction with this to get the arguments
+const ARGUMENTMASK: u16 = 0b0000111111111111;
```

Comment:
Writing the binary as `0b_0000_0000_0000_0000` can be easier to read for you, when debugging and for others

Reply:
I didn't know this! 

Great tip. 
Thank you Lean ^_^